config/finders/all_content.yml

```diff
@@ -55,19 +51,15 @@ details:
     type: date
   show_summaries: true
   sort:
-  - key: "-popularity"
+  - default: true
+    key: "-popularity"
     name: Most viewed
   - key: "-relevance"
     name: Relevance
-  - default: true
-    key: "-public_timestamp"
+  - key: "-public_timestamp"
     name: Updated (newest)
   - key: public_timestamp
     name: Updated (oldest)
 routes:
 - path: "/all-content"
   type: exact
-- path: "/all-content.atom"
-  type: exact
```

Comment:
Will this also remove the route, or is that a separate task?

Reply:
I suspect it will remove the route through the publishing api, but I'll make a note to check.